clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -1035,7 +1035,6 @@ void Clang::AddPreprocessingOptions(Compilation &C, const JobAction &JA,
                                     const InputInfo &Output,
                                     const InputInfoList &Inputs) const {
   const bool IsIAMCU = getToolChain().getTriple().isOSIAMCU();
-  const bool IsIntelFPGA = Args.hasArg(options::OPT_fintelfpga);
```

Comment:
We need to visit SPIR-V cleanup some time:
https://github.com/intel/llvm/blob/fb70c93770981a9ea1ae74d8541de56a150078c9/clang/lib/Driver/ToolChains/Clang.cpp#L10604